libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineApi.kt

```diff
@@ -366,7 +366,7 @@ class MapboxRouteLineApi(
     ) {
         ThreadController.getMainScopeAndRootJob().scope.launch {
             val state = findClosestRoute(target, mapboxMap, padding)
-            resultConsumer.accept(state)
```

Comment:
"accept" is consistent with the Java naming convention for the consumer interface. What was the motivation for changing the name to consume?

Reply:
`consume` sounded more in line with `Consumer`. 🤷 
Will put it back though.